033-OSSDatabaseMigration/Student/00-prereqs.md

```diff
@@ -0,0 +1,58 @@
+# Challenge 0: Pre-requisites - Ready, Set, GO!
+
+**[Home](../README.md)** - [Next Challenge >](./01-assessment.md)
+
+## Introduction
+
+It's time to set up what you will need in order to do these challenges for OSS DB migration
+
+## Description
+
+In this challenge you'll be setting up your environment so that you can complete your challenges.
+
+- Make sure that you have joined the Microsoft Teams group for this track. The first person on your team at your table should create a new channel in this Team with your team name.
```

Comment:
If an attendee hasn't joined the Team for this hack, they likely will never have seen this text! ;)  This should be standard boiler plate lecture from the WTH KickOff template deck.  It's redundant to keep here.

Reply:
Removed the text